<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-P7VKIKU" name="Test" tests="3" failures="1" timestamp="20 Apr 2018 18:49:51 GMT" time="8.019" errors="0">
  <testcase name="teste01" time="1.413" classname="teste.Testar"/>
  <testcase name="teste02" time="1.29" classname="teste.Testar"/>
  <testcase name="teste03" time="1.21" classname="teste.Testar">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at teste.Testar.teste03(Testar.java:66)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- teste03 -->
</testsuite> <!-- Test -->
